src/routes/formsgSiteLaunch.ts

```diff
@@ -224,52 +222,118 @@ export class FormsgSiteLaunchRouter {
+      </tr>`
+    })
+    html += `
+    </tbody></table>
+    <p>This email was sent from the Isomer CMS backend.</p>`
+
+    await mailer.sendMail(email, subject, html)
   }
 
-  sendVerificationDetails = async (
-    requestorEmail: string,
-    repoName: string,
+  sendDNSRecords = async (
     submissionId: string,
-    domainValidationSource: string,
-    domainValidationTarget: string,
-    primaryDomainSource: string,
-    primaryDomainTarget: string,
-    redirectionDomain?: RedirectionDomainObject
+    dnsRecordsEmailProps: DnsRecordsEmailProps[]
   ): Promise<void> => {
-    const subject = `[Isomer] Launch site ${repoName} domain validation`
-    let html = `<p>Isomer site ${repoName} is in the process of launching. (Form submission id [${submissionId}])</p>
-<p>Please set the following CNAME record:</p>
-<p>Source: ${domainValidationSource}</p>
-<p>Target: ${domainValidationTarget}</p>
-<p>Source: ${primaryDomainSource}</p>
-<p>Target: ${primaryDomainTarget}</p>`
-    if (redirectionDomain) {
-      html += `<p>Source: ${redirectionDomain.source}</p>\n<p>Target: ${redirectionDomain.target}</p>\n`
-    }
+    if (dnsRecordsEmailProps.length === 0) return
+    const { requesterEmail } = dnsRecordsEmailProps[0]
+    const subject = `[Isomer] DNS records for launching websites`
+
+    let html = `<p>Isomer sites are in the process of launching. (Form submission id [${submissionId}])</p>
+    <table>
+    <thead>
+      <tr>
+        <th>Repo Name</th>
+        <th>Source</th>
+        <th>Target</th>
+        <th>Type</th>
+      </tr>
+    </thead>
+    <tbody>`
+    dnsRecordsEmailProps.forEach((dnsRecords) => {
+      // check if dnsRecords.redirectionDomain is undefined
+      const isRedirection = !!dnsRecords.redirectionDomain
+      html += `
+    <tr>
+      <td>${dnsRecords.repoName}</td>
+      <td>${dnsRecords.domainValidationSource}</td>
+      <td>${dnsRecords.domainValidationTarget}</td>
+      <td>CNAME</td>
+    </tr>
+    <tr>
+      <td>${dnsRecords.repoName}</td>
+      <td>${
+        isRedirection
+          ? // if redirection, website will be hosted in the 'www' subdomain
+            `www.${dnsRecords.primaryDomainSource}`
+          : dnsRecords.primaryDomainSource
+      }</td>
+      <td>${dnsRecords.primaryDomainTarget}</td>
+      <td>CNAME</td>
+    </tr>`
+
+      if (isRedirection) {
+        html += `
+      <tr>
+        <td>${dnsRecords.repoName}</td>
+        <td>${dnsRecords.redirectionDomain?.source}</td>
+        <td>${dnsRecords.redirectionDomain?.target}</td>
+        <td>A Record</td>
+      </tr>`
+      }
+    })
+    html += `</tbody></table>
+    <p>This email was sent from the Isomer CMS backend.</p>`
+    await mailer.sendMail(requesterEmail, subject, html)
+  }
 
-    html += `<p>This email was sent from the Isomer CMS backend.</p>`
-    await mailer.sendMail(requestorEmail, subject, html)
+  private async siteLaunchHandler(
+    formResponses: FormResponsesProps[],
```

Comment:
how do we poll for `formResponses` btw? does this get processed results too? 

eg: 
1. form has response R1
2. polls and gets [R1]
3. does message sending
4. form has responses [R1, R2]
5. polls and gets [R1, R2]
6. sends [R1, R2]

Reply:
Nope, the results of the site launch (ie whether or not the site is now live or not) are still emailed individually. While one could tackle that for neatness, I think this is ok for now since it has relatively lower impact. The bigger pain point that ops has is sending over the DNS records.

Reply:
just double checking cos the primary motive for this PR was to ease ops searching for details - the success/failure details are in the email title right, so there's minimal manual work required to see the site details?

if so, please feel free to resolve this

Reply:
Not quite haha, there are two different problems
1) Email to Ops regarding DNS records for agencies to put in 
2) Email to Ops whether the site has been successfully launched 
This PR tackles the first problem, the second one requires more effort since we need to parse the queue messages and figure out which site launches were submitted using the same from. Since that would be out of the scope of this PR, resolving this ya